ReferenceError: getReactVersion is not defined

I try to unlink react-native-camer, but I get this error : 
```
$ rnpm unlink react-native-camera
rnpm info Unlinking react-native-camera android dependency
/Users/jbertrand/.nvm/versions/node/v5.6.0/lib/node_modules/rnpm/node_modules/rnpm-plugin-link/src/android/unregisterNativeModule.js:11
  const prefix = getPrefix(getReactVersion(projectConfig.folder));
                           ^

ReferenceError: getReactVersion is not defined
    at unregisterNativeAndroidModule (/Users/jbertrand/.nvm/versions/node/v5.6.0/lib/node_modules/rnpm/node_modules/rnpm-plugin-link/src/android/unregisterNativeModule.js:11:28)
    at Object.unlink [as func] (/Users/jbertrand/.nvm/versions/node/v5.6.0/lib/node_modules/rnpm/node_modules/rnpm-plugin-link/src/unlink.js:42:30)
    at Command.runAction (/Users/jbertrand/.nvm/versions/node/v5.6.0/lib/node_modules/rnpm/bin/cli:24:15)
    at Command.listener (/Users/jbertrand/.nvm/versions/node/v5.6.0/lib/node_modules/rnpm/node_modules/commander/index.js:301:8)
    at emitTwo (events.js:100:13)
    at Command.emit (events.js:185:7)
    at Command.parseArgs (/Users/jbertrand/.nvm/versions/node/v5.6.0/lib/node_modules/rnpm/node_modules/commander/index.js:615:12)
    at Command.parse (/Users/jbertrand/.nvm/versions/node/v5.6.0/lib/node_modules/rnpm/node_modules/commander/index.js:458:21)
    at Object.<anonymous> (/Users/jbertrand/.nvm/versions/node/v5.6.0/lib/node_modules/rnpm/bin/cli:30:5)
    at Module._compile (module.js:413:34)
```

I tried to add `const getReactVersion = require('../getReactNativeVersion');` at line 6 of `rnpm-plugin-link/src/android/unregisterNativeModule.js` and I get this error : 
```
$ rnpm unlink react-native-camera
rnpm info Unlinking react-native-camera android dependency
module.js:341
    throw err;
    ^

Error: Cannot find module './patches/0.18/addPackagePath'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at unregisterNativeAndroidModule (/Users/jbertrand/.nvm/versions/node/v5.6.0/lib/node_modules/rnpm/node_modules/rnpm-plugin-link/src/android/unregisterNativeModule.js:32:30)
    at Object.unlink [as func] (/Users/jbertrand/.nvm/versions/node/v5.6.0/lib/node_modules/rnpm/node_modules/rnpm-plugin-link/src/unlink.js:42:30)
    at Command.runAction (/Users/jbertrand/.nvm/versions/node/v5.6.0/lib/node_modules/rnpm/bin/cli:24:15)
    at Command.listener (/Users/jbertrand/.nvm/versions/node/v5.6.0/lib/node_modules/rnpm/node_modules/commander/index.js:301:8)
    at emitTwo (events.js:100:13)
    at Command.emit (events.js:185:7)
```

After some research I found that there is a file called `addPackagePatch` (and NOT addPackagePath) so I tried to change the name but I get this error : 
```
$ rnpm unlink react-native-camera
rnpm info Unlinking react-native-camera android dependency
/Users/jbertrand/.nvm/versions/node/v5.6.0/lib/node_modules/rnpm/node_modules/rnpm-plugin-link/src/android/patches/0.18/addPackagePatch.js:2
  ',\n        ' + config.packageInstance;
                        ^

TypeError: Cannot read property 'packageInstance' of undefined
    at module.exports (/Users/jbertrand/.nvm/versions/node/v5.6.0/lib/node_modules/rnpm/node_modules/rnpm-plugin-link/src/android/patches/0.18/addPackagePatch.js:2:25)
    at /Users/jbertrand/.nvm/versions/node/v5.6.0/lib/node_modules/rnpm/node_modules/rnpm-plugin-link/src/android/unregisterNativeModule.js:67:35
    at /Users/jbertrand/.nvm/versions/node/v5.6.0/lib/node_modules/rnpm/node_modules/lodash.flowright/index.js:38:31
    at unregisterNativeAndroidModule (/Users/jbertrand/.nvm/versions/node/v5.6.0/lib/node_modules/rnpm/node_modules/rnpm-plugin-link/src/android/unregisterNativeModule.js:71:8)
    at Object.unlink [as func] (/Users/jbertrand/.nvm/versions/node/v5.6.0/lib/node_modules/rnpm/node_modules/rnpm-plugin-link/src/unlink.js:42:30)
    at Command.runAction (/Users/jbertrand/.nvm/versions/node/v5.6.0/lib/node_modules/rnpm/bin/cli:24:15)
    at Command.listener (/Users/jbertrand/.nvm/versions/node/v5.6.0/lib/node_modules/rnpm/node_modules/commander/index.js:301:8)
    at emitTwo (events.js:100:13)
    at Command.emit (events.js:185:7)
    at Command.parseArgs (/Users/jbertrand/.nvm/versions/node/v5.6.0/lib/node_modules/rnpm/node_modules/commander/index.js:615:12)
```

And I'm stuck here :disappointed:

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "rnpm-plugin-link",
-  "version": "1.5.3",
+  "version": "1.5.4",
   "description": "rnpm plugin that links native dependencies to your project",
   "main": "index.js",
   "scripts": {

--- a/src/android/unlinkAssets.js
+++ b/src/android/unlinkAssets.js
@@ -13,6 +13,8 @@ module.exports = function unlinkAssetsAndroid(files, targetPath) {
 
   grouped.font.forEach((file) => {
     const filename = path.basename(file);
-    fs.unlinkSync(path.join(targetPath, 'fonts', filename));
+    if (fs.existsSync(filename)) {
+      fs.unlinkSync(path.join(targetPath, 'fonts', filename));
+    }
   });
 };

--- a/src/android/unregisterNativeModule.js
+++ b/src/android/unregisterNativeModule.js
@@ -2,10 +2,11 @@ const readFile = require('./fs').readFile;
 const writeFile = require('./fs').writeFile;
 const path = require('path');
 const compose = require('lodash.flowright');
+const getReactVersion = require('../getReactNativeVersion');
 const getPrefix = require('./getPrefix');
 
 const cut = (scope, pattern) =>
-  scope.replace(pattern + '\n', '');
+  scope.replace(pattern, '');
 
 module.exports = function unregisterNativeAndroidModule(name, dependencyConfig, projectConfig) {
   const prefix = getPrefix(getReactVersion(projectConfig.folder));
@@ -17,7 +18,7 @@ module.exports = function unregisterNativeAndroidModule(name, dependencyConfig, 
   const cutModuleFromSettings = (name) => (content) =>
     cut(content, `include ':${name}'\n` +
       `project(':${name}').projectDir = ` +
-      `new File(rootProject.projectDir, '../node_modules/${name}/android')`
+      `new File(rootProject.projectDir, '../node_modules/${name}/android')\n`
     );
 
   /**
@@ -26,9 +27,9 @@ module.exports = function unregisterNativeAndroidModule(name, dependencyConfig, 
    * @return {String}         Patched content of Build.gradle
    */
   const cutModuleFromBuild = (name) => (content) =>
-    cut(content, `    compile project(':${name}')`);
+    cut(content, `    compile project(':${name}')\n`);
 
-  const getAddPackagePatch = require(`./${prefix}/addPackagePath`);
+  const getAddPackagePatch = require(`./${prefix}/addPackagePatch`);
 
   /**
    * Make a MainActivity.java program patcher
@@ -37,8 +38,8 @@ module.exports = function unregisterNativeAndroidModule(name, dependencyConfig, 
    * @return {Function}            Patcher function
    */
   const makeMainActivityPatcher = (content) => {
-    const patched = cut(content, dependencyConfig.packageImportPath);
-    return cut(patched, getAddPackagePatch());
+    const patched = cut(content, dependencyConfig.packageImportPath + '\n');
+    return cut(patched, getAddPackagePatch(dependencyConfig));
   };
 
   const applySettingsGradlePatch = compose(
@@ -63,7 +64,7 @@ module.exports = function unregisterNativeAndroidModule(name, dependencyConfig, 
    * Check if module has been installed already
    */
   const isInstalled = compose(
-    (content) => ~content.indexOf(getAddPackagePatch()),
+    (content) => ~content.indexOf(getAddPackagePatch(dependencyConfig)),
     readFile(projectConfig.mainActivityPath)
   );
 

